lib/_sidebar.erb

```diff
@@ -13,14 +13,28 @@
                 <%- else -%>
                   <%- active_keyword="" -%>
                 <%- end -%>
-                <li class="<%=active_keyword%>"><a href="<%=key2["slug"]%>.html" data-toggle="tab"><%=key2["title"]%></a></li>
+                <li class="<%=active_keyword%>"><a href="#<%=key2["slug"]%>" data-toggle="tab"><%=key2["title"]%></a></li>
               <%- end -%>
             <%- end -%>
           </ul>
         </nav>
       </div>
       <!-- Tab content -->
       <div class="tab-content">
-        <div class="tab-pane active text-style" id="content"></div>
+      <%- @site_map['docs'].each do |key, value| -%>
+        <%- first_entry = true -%>
+        <%- value.each do |key2| -%>
```

Comment:
`.keys` is missing.
`value.each do |key2|` => `value.keys.each do |key2|`

Reply:
I think they have the same effect. I will check.